### DISCUSSION TITLE
Can sponsors request re-prioritization of releases?

### DISCUSSION THREAD

**Discussion by psifertex** [squidfunk/mkdocs-material#3973] [Q&A] [answered]
I'm considering (still doing some testing but overall really pleased with the results so far) moving over my commercial product's [documentation](https://github.com/Vector35/binaryninja-api/tree/dev/docs) to mkdocs-material from the mkdocs RTD theme and am happy to sponsor. Unfortunately, one thing that holds me back is we've got build infrastructure that we try to keep as simple as possible and switching over to using a private repo for insiders would complicate things in an undesirable way. 

Is it possible to request re-ordering of a feature before sponsorship? I'd love to be the one who pushes over the limit to $6k for the next tranche of features, but the feature I'm most interested in is offline support which is not currently scheduled until $10k/mo which is another year and a half away.

**Comment by squidfunk** [comment, marked as answer]
Thanks for considering sponsoring! Unfortunately, reprioritization of features is not something to be considered. The tier structure is not arbitrary, a lot of thinking went into it. For this reason, offline support will stay within the $10k goal.

However, setting up Insiders in CI is not hard. It boils down to creating a bot account with an access token:
https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions

**Reply by psifertex** [reply]
I realize in isolation it's not much work, but considering we have multiple build machines across multiple architectures we work hard to try to keep the requirements for our internal CI relatively low. It's definitely not a hard-block but just a minor inconvenience, but enough paper cuts can add up when you're juggling a lot of dependencies.

Totally understand about the feature planning. Coding is easier than build infrastructure, which is easier than feature selection, which is easier than pricing in my experience. 😉 

We honestly might just sponsor and still choose to use the public release, we'll see. Thanks for the quick response!

**Reply by squidfunk** [reply]
You probably have private repositories anyway – we also offer a Docker image you can build and push to a private registry. Alternatively, you can fork the Insiders repository and install it alongside other private/shared repositories in your organization.

There are many possibilities for installing and deploying Material for MkDocs 😊 Nonetheless, thanks for considering sponsoring!
